include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -9957,6 +9957,48 @@
           "extensions" : [ "SPV_EXT_shader_stencil_export" ],
           "version" : "None"
         },
+        {
+          "enumerant": "StencilRefUnchangedFrontAMD",
+          "value": 5079,
+          "capabilities": [ "StencilExportEXT" ],
+          "extensions": [ "SPV_AMD_fragment_discard_storage_writes", "SPV_EXT_shader_stencil_export" ],
```

Comment:
nit: all `SPV_AMD_fragment_discard_storage_writes` should be `SPV_AMD_shader_early_and_late_fragment_tests`